Framework/PythonInterface/plugins/algorithms/MuonMaxent.py

```diff
@@ -220,32 +227,69 @@ def getPhase(self,FLAGS_fixphase,POINTS_ngroups,POINTS_nhists,mylog):
             names = pt.getColumnNames()
             phaseLabel = None
             IDLabel = None
+            asymmLabel = None
             for name in names:
```

Comment:
We've got loads of calls to `lower` and string comparisons here, which has the potential to get a bit unwieldy. It would be a bit faster (and to my eye more readable) to do:
```
name_lower = name.lower()
if name_lower in {"phi", "phase"}:
    phaseLabel = name
elif name_lower in {"detid", "spectrum number"}:
    IDLabel = name
elif name_lower in {"asymmetry", "asymm", "asasym"}:
    asymmLabel = name

Reply:
yes that is nicer

Reply:
spotted a typo